client/src/containers/requester/role/MemberList.js

```diff
@@ -110,14 +109,16 @@ class MemberList extends Component {
         <Grid.Column key={userId} largeScreen={8} widescreen={5}>
           <Segment className='avatar no-padding minimal'>
             { isOwner ?
-              <Icon
+              <Popup trigger={<Icon
                 name='shield'
                 className='pull-right'
-                color='green'/> :
-              <Icon
+                color='green'/>} content='Owner' id='owner-popup-box'
```

Comment:
IDs should be prepended with `next-` for specific styles to avoid collisions with vendor rules. For example, `next-member-list-owner-popup-box`.